daemon/src/maker_cfd_actor.rs

```diff
@@ -416,4 +537,12 @@ impl Message for SyncWallet {
     type Result = ();
 }
 
+impl Message for AcceptOrder {
+    type Result = Result<()>;
```

Comment:
Use `()` please and wrap all the functions in handlers that use the `log_error!` macro, otherwise we the errors will be silently dropped.